.gitignore

```diff
@@ -24,3 +24,9 @@ dashboard/app/.gcloudignore
 
 # jetbrains goland
 .idea
+
+# VM related stuff
+buildroot-amd64-2020.11
```

Comment:
I would suggest to just put these into some other dir. We can't add all files here that somebody decided to store in the checkout dir :)
entitlements.xml look different, what is it? Is it related to some editor?

Reply:
It's overriding some hypervisor-related flag in MacOS for qemu. The .gitignore file was committed accidentaly, sorry!